vibi-dpu/src/core/coverage.rs

```diff
@@ -14,24 +15,32 @@ pub async fn process_coverage(hunkmap: &HunkMap, review: &Review) {
     for prhunk in hunkmap.prhunkvec() {
         // calculate number of hunks for each userid
         let coverage_map = calculate_coverage(&hunkmap.repo_owner(), prhunk);
+        let coverage_cond = !coverage_map.is_empty();
+        println!("!coverage_map.is_empty() = {:?}", &coverage_cond);
+        println!("repo_config.comment() = {:?}", repo_config.comment());
+        println!("repo_config.auto_assign() = {:?}", repo_config.auto_assign());
         if !coverage_map.is_empty() {
```

Comment:
No never nesting?

Reply:
Created an issue, will raise another PR

Reply:
Created new issue, will raise separate PR for this